react-props-and-expressions/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,9 @@
+function CustomButton({ text, color }) {
+  const style = {
+    backgroundColor: color,
+  };
+
```

Comment:
depending on how long your variables get you can decide to inline style as well. 

`return <button style={{
    backgroundColor: color
  }}>{text}</button>;`

Reply:
Oh! Okay I was wondering about that. Thank you!